fuse_ll/src/fuse/mount.rs

```diff
@@ -8,23 +8,126 @@ use std::os::raw::c_void;
 use std::os::unix::io::RawFd;
 use std::path::Path;
 
+use common_param::*;
 use param::*;
 
+/// Mount option validator function
+pub fn options_validator(option: String) -> Result<(), String> {
+    let ops = option.split(',');
+    for op in ops {
+        if !VALID_ON_OFF_OPTIONS.contains(&op) && !key_value_validator(&op) {
```

Comment:
add comment about this if condition, what's the true case and what's the false case?